packages/mui-material/src/StepLabel/StepLabel.test.js

```diff
@@ -86,6 +86,17 @@ describe('<StepLabel />', () => {
   });
 
   describe('<Step /> prop: active', () => {
+    it('renders <StepLabel> with the className active', () => {
+      const { container } = render(
+        <Step active>
+          <StepLabel>Step One</StepLabel>
+        </Step>,
+      );
+
+      const root = container.querySelector(`.${classes.root}`);
+      expect(root).to.have.class(classes.active);
```

Comment:
Isn't this already tested above in `Step.test.js`? I recommend we keep it one place.